src/tasks/components/TaskHeaderCard.tsx

```diff
@@ -0,0 +1,162 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect, ConnectedProps} from 'react-redux'
+import {withRouter, RouteComponentProps} from 'react-router-dom'
+import moment from 'moment'
+
+// Components
+import {
+  SlideToggle,
+  ComponentSize,
+  ResourceCard,
+  InputLabel,
+  FlexBox,
+  AlignItems,
+  FlexDirection,
+  JustifyContent,
+} from '@influxdata/clockface'
+
+// Actions
+import {
+  addTaskLabel,
+  deleteTaskLabel,
+  runTask,
+  getRuns,
+  // setTaskRunsPageAsCurrent,
+} from 'src/tasks/actions/thunks'
+
+// Types
+import {ComponentColor, Button} from '@influxdata/clockface'
+import {Task, AppState} from 'src/types'
+
+interface PassedProps {
+  task: Task
+  onActivate: (task: Task) => void
+  onEditTask: () => void
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = PassedProps & ReduxProps
+
+export class TaskHeaderCard extends PureComponent<
+  Props & RouteComponentProps<{orgID: string; id: string}>
+> {
+  public render() {
+    const {task} = this.props
+
+    if (!task) {
+      return null
+    }
+
+    return (
+      <ResourceCard
+        testID="task-card"
```

Comment:
you probably want to scope this a little tighter, something like

```
"task-runs-task-card"
```

to provide semantic meaning that it's a task card on the task runs page